utils/bootstrap-node

```diff
@@ -207,6 +207,10 @@ wait_m0mkfs_done() {
     sleep 2
 }
 
+if [ -z $IDs ]; then
+    echo -n " Service not found "
```

Comment:
Which service not found?

Reply:
Done, removed it, since it was misleading.